tests/unit/test_models/test_submodels/test_porosity/test_leading_reaction_driven_porosity.py

```diff
@@ -9,14 +9,24 @@
 
 class TestLeadingOrder(unittest.TestCase):
     def test_public_functions(self):
-        param = pybamm.LeadAcidParameters()
+        # param = pybamm.LeadAcidParameters()
+        param = pybamm.LithiumIonParameters()
```

Comment:
Not sure if we should still test with `LeadAcidParameters` too or not. @tinosulzer?

Reply:
It's probably why we miss the coverage earlier but it's fine - need to reformat these porosity models anyway